graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsSchemaProvider.kt

```diff
@@ -430,23 +435,17 @@ class DgsSchemaProvider(
                 logger.info("No schema files found, but a schema was provided as an TypeDefinitionRegistry")
                 arrayOf()
             } else {
-                logger.error("No schema files found. Define schemas in src/main/resources/$basedir/**/*.graphqls")
+                logger.error("No schema files found. Define schema locations with property dgs.graphql.schema-locations")
                 throw NoSchemaFoundException()
             }
         }
 
-        val testSchemas = try {
-            resolver.getResources("classpath:$basedir-test/**/*.graphql*")
-        } catch (ex: Exception) {
-            arrayOf<Resource>()
-        }
-
         val metaInfSchemas = try {
-            resolver.getResources("classpath*:META-INF/$basedir/**/*.graphql*")
+            resolver.getResources("classpath*:META-INF/schema/**/*.graphql*")
```

Comment:
Should we make meta-inf configurable as well? Or will it always be under `/schema` ?

Reply:
Yes, this is related to my own comment in the PR and also why 1 test is failing atm (easy fixable though)
I think `META-INF/schema` is needed for the dgs graphql error types schema files.

Reply:
Would removing the test and meta-inf schemas be a breaking change? I'm not super familiar w/ their usage to make a good call on that. 

However, if we end up keeping parity w/ current impl we could just create 2 more properties for the locations and default them to what they currently are set to. This would in effect solve the "configure the locations" ask. We could follow on w/ subsequent MR w/ ability to opt-out of these if desired.

Reply:
To be honest adding the test schema doesn't really make sense imo - I also couldn't find any failing tests because of that.
I of course don't know if this is used internally at Netflix as I assume it is coming from somewhere. Anyway like I said, when needed, users can add that location to ` dgs.graphql.schema-locations`, even only for a Spring test profile.
But yes, it might be  breaking change.

For the META-INF schemas I think we first need to know if they are required to be added or not. Maybe @paulbakker knows?

One of the risks I see is that adding `classpath*:META-INF/schema/**/*.graphql*` always is that it could lead to unexpected behaviour when another project dependency puts graphql files in that folder. I think the risk is very low btw ;-)

Maybe instead `classpath*:META-INF/dgs/schema/**/*.graphql* could be added?
It means moving these schema's into a different folder (`dgs`), but that way DGS can always contribute schema parts. Again this is not 100% safe, but imo a good trade off.

Reply:
All good points. Let's see what Paul and @berngp enlighten us around the usage of these.

Reply:
The meta-info/schema mechanism is so that you can provide common types in a library. E.g. internally we have a library with all the common scalars we use in our federated graph. So, the meta-inf/schema mechanism is important and should stay. Since we already have these libraries published we also can't change the path inside META-INF.

The test-schemas thing I honestly can't remember what it was for (although I'm sure I was the one who added it...). Now that the path is configurable I'm ok removing that.

Reply:
Got it. Yeh I ran into the whole "required open" thing when considering adding validation to the config props via JSR-303. Because data classes can't be open but Spring AOP needed them to be open I got it working w/ the `all-open` plugin. I am sure you have already seen the [kotlin-spring](https://kotlinlang.org/docs/all-open-plugin.html#spring-support) variant of it but are  just steering away from plugins in general.

Reply:
@paulbakker @bono007  I understand about the META-INF/schema and then let's keep that fixed path always to be included.

I updated the failing test and I think the PR is good for a final review.